pkg/users/account-create-dialog.js

```diff
@@ -167,22 +169,44 @@
     return remove_diacritics(result);
 }
 
-export function account_create_dialog(accounts) {
+export function account_create_dialog(accounts, shells) {
     let dlg = null;
     const state = {
+        dialogLoading: true,
         real_name: "",
         user_name: "",
         password: "",
         password_confirm: "",
         locked: false,
         confirm_weak: false,
         change_passw_force: false,
+        base_home_dir: null,
+        shell: null,
     };
     let errors = { };
 
     let old_password = null;
     let user_name_dirty = false;
 
+    function get_defaults() {
+        return cockpit.spawn(["useradd", "-D"], { superuser: "require", err: "message" })
+                .catch(e => console.warn("Could not get useradd defaults: ", e.message))
+                .then(defaults => {
+                    let shell = null;
+                    let base_home_dir = null;
+                    defaults.split("\n").forEach(item => {
+                        if (item.indexOf("SHELL=") === 0) {
+                            shell = item.split("=")[1] || "/bin/bash";
+                        } else if (item.indexOf("HOME=") === 0) {
+                            base_home_dir = item.split("=")[1] || "";
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18471-20230316-000158-3ca534fe-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)